connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java

```diff
@@ -68,7 +68,7 @@ public void translate(MovePlayerPacket packet, GeyserSession session) {
         double javaY = packet.getPosition().getY() - EntityType.PLAYER.getOffset();
         if (packet.isOnGround()) javaY = Math.ceil(javaY * 2) / 2;
         ClientPlayerPositionRotationPacket playerPositionRotationPacket = new ClientPlayerPositionRotationPacket(
-                packet.isOnGround(), GenericMath.round(packet.getPosition().getX(), 4), javaY, GenericMath.round(packet.getPosition().getZ(), 4), packet.getRotation().getY(), packet.getRotation().getX()
+                packet.isOnGround(), Double.parseDouble(Float.toString(packet.getPosition().getX())), javaY, Double.parseDouble(Float.toString(packet.getPosition().getZ())), packet.getRotation().getY(), packet.getRotation().getX()
```

Comment:
Can you add a comment explaining why we're converting from a float to string to double?

Reply:
Done.